.github/workflows/release.yml

```diff
@@ -56,13 +56,24 @@ jobs:
       - name: Building packages
         run: yarn build
 
-      - name: Create Release Pull Request or Publish to npm
+      - name: Storing release version for changeset
+        id: release_version
+        run: echo "::set-output name=value::$(./scripts/print_release_version.sh)"
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
```

Comment:
We need to "extract" the "next" release version before the `changesets/action` step, as the value is used for the `githubReleaseName` and `githubTagName` config values.